src/Pages/Login/Login.js

```diff
@@ -69,39 +72,90 @@ export default class extends Component {
       })
   }
 
+  updateCSS (opt) {
+    opt === 1
+      ? this.setState({
+        AnimationCSS1: {
+          top: '-5px',
+          borderBottom: '2px solid GREEN'
+        }
+      })
+      : this.setState({ AnimationCSS1: {} })
+
+    opt === 2
+      ? this.setState({
+        AnimationCSS2: {
+          top: '-5px',
+          borderBottom: '2px solid GREEN'
+        }
+      })
+      : this.setState({ AnimationCSS2: {} })
+  }
+
+  inputBox (field) {
+    return (
+      <div
+        style={field.style}
+        onFocus={() => {
+          this.updateCSS(field.css)
+        }}
+        onBlur={() => {
+          this.updateCSS()
+        }}
+        className='txtb'
+      >
+        <span />
+        <input
+          type={field.type}
+          name={field.type}
+          placeholder={field.placeholder}
+          value={field.value}
+          onChange={this.handleChange}
+          required
+        />
+      </div>
+    )
+  }
+
   render () {
-    const { email, password, message } = this.state
+    const { message } = this.state
```

Comment:
for every this.state, thow it into the bracekts on line 121.

e.g.

const { message, AnimationCSS1, email, ... } = this.state;